cea/demand/thermal_loads.py

```diff
@@ -1148,12 +1148,12 @@ def get_temperatures(locator, prop_HVAC):
     :type prop_HVAC: Gdf
 
     Sample data (first 5 rows):
-                 Name type_cs type_ctrl type_dhw type_hs
-    0     B154862      T0        T1       T1      T1
-    1     B153604      T0        T1       T1      T1
-    2     B153831      T0        T1       T1      T1
-    3  B302022960      T0        T0       T0      T0
-    4  B302034063      T0        T0       T0      T0
+                 Name type_cs type_ctrl type_dhw type_hs MECH_VENT WIN_VENT HEAT_REC NIGHT_FLSH ECONOMIZER
```

Comment:
@JIMENOFONSECA : But how are heating and cooling emission systems related to ventilation systems?
I don't think this is a good idea, as this would just multiply the number of keys, because every possible configuration has to be represented by an own key (e.g. heating with radiators, no cooling, mechanical ventilation, with heat exchanger, without economizer, without night flushing) would be one single key e.g. T54. In that way nobody will no what is going on.

I agree that the new columns have to be added to the documentation. I will create an issue for that and start working on that after the changes are final.